doc/tutorial/remote-apis.md

```diff
@@ -0,0 +1,105 @@
+# Remote APIs
+
+JSTP allows you to use remote APIs as regular asynchronous functions, let's
+examine how it looks like by updating our `client.js` file like this:
+
+```javascript
+'use strict';
+
+const jstp = require('@metarhia/jstp');
+
+const APP_NAME = 'chatApp';
+const PORT = 3000;
+const HOSTNAME = '127.0.0.1';
+
+jstp.ws.connectAndInspect(
+  APP_NAME,
+  null,
+  ['auth', 'chat'],
+  {},
+  `ws://${HOSTNAME}:${PORT}`,
+  (error, connection, app) => {
+    if (error) {
+      console.error(error);
+      return;
+    }
+
+    console.log(`Connected to ${APP_NAME}`);
+
+    const username = 'user_1';
+    const password = 'password_1';
+
+    app.auth.register(username, password, error => {
+      if (error) {
+        console.error(error);
+        return;
+      }
+
+      console.log('Registered');
+
+      app.auth.login(username, password, error => {
+        if (error) {
+          console.error(error);
+          return;
+        }
+
+        console.log(`Logged in as ${username}`);
+      });
+    });
+  }
+);
+```
+
+> The code added in callback at this step works both in Node.js and browser, so
+> further we will consider only example in Node.js.
+
+We've just added some output (all that `console.log()` debug) to understand,
+what steps we've proceeded so far. Then, of course, we handle errors, which can
+appear during an interaction. So you can see, that you can call remote methods
+from interfaces like regular asynchronous functions in your process. `app`
```

Comment:
`As you can see, remote methods call simply be called as regular asynchronous functions.`

Reply:
Without `simply`, I suppose, as @belochub has drawn attention on this earlier.

Reply:
Ok, and call -> can )